execution/gethexec/sequencer.go

```diff
@@ -67,9 +67,10 @@ type SequencerConfig struct {
 	NonceFailureCacheSize       int             `koanf:"nonce-failure-cache-size" reload:"hot"`
 	NonceFailureCacheExpiry     time.Duration   `koanf:"nonce-failure-cache-expiry" reload:"hot"`
 	// Espresso specific flags
-	Espresso          bool   `koanf:"espresso"`
-	HotShotUrl        string `koanf:"hotshot-url"`
-	EspressoNamespace uint64 `koanf:"espresso-namespace"`
+	Espresso             bool   `koanf:"espresso"`
+	HotShotUrl           string `koanf:"hotshot-url"`
+	EspressoNamespace    uint64 `koanf:"espresso-namespace"`
+	EspressoContractAddr string `koanf:"espresso-contract-addr"`
```

Comment:
Remove this ContractAddr?

Reply:
I was not sure if we needed this. I will remove it